apps/central-scan/frontend/src/app_root.tsx

```diff
@@ -80,48 +80,6 @@ export function AppRoot({
   const { adjudication } = status;
 
   const [isScanning, setIsScanning] = useState(false);
-  const [currentScanningBatchId, setCurrentScanningBatchId] =
-    useState<string>();
-  const [lastScannedSheetIdx, setLastScannedSheetIdx] = useState(0);
-
-  useEffect(() => {
-    if (!currentScanningBatchId) {
-      return;
-    }
-    const currentBatch = status.batches.find(
-      (b) => b.id === currentScanningBatchId
-    );
-    if (!currentBatch) {
-      return;
-    }
-    if (currentBatch.count > lastScannedSheetIdx) {
-      if (status.adjudication.remaining > 0) {
-        // Scanning a sheet failed and needs adjudication.
-        void logger.log(LogEventId.ScanSheetComplete, userRole, {
-          disposition: 'failure',
-          message: 'Sheet rejected while scanning.',
-          result:
-            'Sheet not tabulated, user asked to intervene in order to proceed.',
-          batchId: currentScanningBatchId,
-          sheetCount: currentBatch.count,
-        });
-        return;
```

Comment:
I opted to just remove this logging because it is always followed by an "AdjudicationInfo" log. The problem is not really that the scan failed, but that the scan went to adjudication.